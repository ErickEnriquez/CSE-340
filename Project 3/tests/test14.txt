{
    y : STRING ;
    x : BOOLEAN ;
    x = TRUE ;
    x = ~ y ;
}