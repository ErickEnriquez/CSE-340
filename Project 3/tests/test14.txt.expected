TYPE MISMATCH 5 C8